src/pages/login/Login.tsx

```diff
@@ -2,18 +2,20 @@ import * as React from 'react';
 import { Col, Container, Row, Spinner } from 'reactstrap';
 import { Redirect, RouteProps } from 'react-router-dom';
 
-import AuthenticationHeader from 'components/work/AuthenticationHeader';
+import AuthHeader from 'components/work/AuthHeader';
+import UserTestimonials from 'components/work/UserTestimonials';
 import { FirebaseConsumer, FirebaseUserProps } from 'HOCs/FirebaseProvider';
+import { AUTHENTICATION_CONTAINER, AUTHENTICATION_CONTENT } from 'styled/sharedClasses/authentication';
+import { VIEWPORT_CENTER_LAYOUT } from 'styled/sharedClasses/layout';
 
 import LoginForm from './LoginForm';
-import LoginTestimonals from './LoginTestimonials';
 
 const Login = (props: RouteProps) => (
   <FirebaseConsumer>
     {({ loading, user }: FirebaseUserProps) => {
       if (loading) {
         return (
-          <Container className="d-flex align-items-center justify-content-center position-relative height-100vh">
+          <Container className={VIEWPORT_CENTER_LAYOUT}>
```

Comment:
The other way of doing this is `<CenterLayoutContainer>` instead of the css classes in a string so the programmer doesn't have to think whether to use a "Container" or a "Row" or a "div".

Unless the intended behavior is that the classes should be applied independent of the component